lighthouse-core/report/v2/report-renderer.js

```diff
@@ -98,19 +102,42 @@ window.ReportRenderer = class ReportRenderer {
    * @param {string} description
    * @return {!Element}
    */
-  _renderScore(score, title, description) {
-    const element = this._createElement('div', 'lighthouse-score');
+  _renderScore(score, title, description, auditOrCategory) {
```

Comment:
can someone sum up what was decided? My thoughts:

- audits are going to score in some fashion, whether that's 0-100, bucketed into grades, or pass/fail. Even if we require all audits to report a score as a number, it's still going to be an integer in that interval, buckets of every 10ish numbers, or 0/100. It makes some sense then for the audit to declare how it scored (perhaps on audit.meta), even if the reporter changes how the score is presented.

- If on the other hand (or in addition to the above), we allow specifying the precision type in the config (on one of the audits in `categories.categoryName.audits`) it seems like we'd need some way of converting the score reported by the audit into the new format. `binary` -> `numeric` is easy, but (as discussed), what's the true/false cutoff for `numeric` -> `binary` and where do you specify it?